features/support/env.rb

```diff
@@ -33,7 +33,7 @@
 # Remove/comment out the lines below if your app doesn't have a database.
 # For some databases (like MongoDB and CouchDB) you may need to use :truncation instead.
 begin
```

Comment:
in the comment says that mongoDB may need to use `:truncation`. Why are we using `:deletion`

Reply:
```ruby
DEPRECATION WARNING: Non-URL-safe CSRF tokens are deprecated. Use 6.1 defaults or above. (called from require at /Users/enrique.silva/Documents/USP/6per/ESI/Kimchi/features/support/env.rb:7)
The 'truncation' strategy does not exist for the mongoid ORM!  Available strategies: deletion (DatabaseCleaner::UnknownStrategySpecified)
/usr/local/lib/ruby/gems/3.1.0/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaner.rb:93:in `rescue in orm_strategy'
/usr/local/lib/ruby/gems/3.1.0/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaner.rb:88:in `orm_strategy'
/usr/local/lib/ruby/gems/3.1.0/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaner.rb:85:in `create_strategy'
/usr/local/lib/ruby/gems/3.1.0/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaner.rb:45:in `strategy='
/usr/local/lib/ruby/gems/3.1.0/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaners.rb:17:in `block in strategy='
/usr/local/lib/ruby/gems/3.1.0/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaners.rb:17:in `each'
/usr/local/lib/ruby/gems/3.1.0/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaners.rb:17:in `strategy='
/usr/local/Cellar/ruby/3.1.2_1/lib/ruby/3.1.0/forwardable.rb:233:in `strategy='
/Users/enrique.silva/Documents/USP/6per/ESI/Kimchi/features/support/env.rb:36:in `<top (required)>'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/glue/registry_and_more.rb:123:in `require'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/glue/registry_and_more.rb:123:in `load_code_file'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/runtime/support_code.rb:145:in `load_file'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/runtime/support_code.rb:82:in `block in load_files!'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/runtime/support_code.rb:81:in `each'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/runtime/support_code.rb:81:in `load_files!'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/runtime.rb:274:in `load_step_definitions'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/runtime.rb:74:in `run!'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/lib/cucumber/cli/main.rb:29:in `execute!'
/usr/local/lib/ruby/gems/3.1.0/gems/cucumber-8.0.0/bin/cucumber:9:in `<top (required)>'
/usr/local/lib/ruby/gems/3.1.0/bin/cucumber:25:in `load'
/usr/local/lib/ruby/gems/3.1.0/bin/cucumber:25:in `<top (required)>'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/cli/exec.rb:58:in `load'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/cli/exec.rb:58:in `kernel_load'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/cli/exec.rb:23:in `run'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/cli.rb:483:in `exec'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/cli.rb:31:in `dispatch'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/cli.rb:25:in `start'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/exe/bundle:48:in `block in <top (required)>'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/lib/bundler/friendly_errors.rb:103:in `with_friendly_errors'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.11/exe/bundle:36:in `<top (required)>'
/usr/local/lib/ruby/gems/3.1.0/bin/bundle:25:in `load'
/usr/local/lib/ruby/gems/3.1.0/bin/bundle:25:in `<main>'
```

Reply:
It seems like there's no `:truncation` strategy for mongoDB, although the suggestion from cucumber doc